pkg/extension/sumologicextension/extension.go

```diff
@@ -34,30 +35,37 @@ type sumologicExtension struct {
 	conf             *Config
 	logger           *zap.Logger
 	registrationInfo OpenRegisterResponsePayload
+	ch               chan struct{}
 }
 
 const (
 	// TODO: fix
-	niteBaseUrl = "https://nite-open-events.sumologic.net/"
+	niteBaseUrl              = "https://nite-open-events.sumologic.net/"
+	heartbeatUrl             = "api/v1/collector/heartbeat"
+	registerUrl              = "api/v1/collector/register"
+	defaultHeartbeatInterval = time.Duration(15)
```

Comment:
15 what? 

```suggestion
	defaultHeartbeatInterval = 15 * time.Second
```

Maybe this?